Gemfile

```diff
@@ -29,7 +29,8 @@ gem 'jbuilder', '~> 2.5'
 
 # Use ActiveStorage variant
 # gem 'mini_magick', '~> 4.8'
-
+gem 'bootstrap', '~> 4.5.0'
```

Comment:
Thought I was going to use this for nav bar, didn't end up using it but might come in handy later on